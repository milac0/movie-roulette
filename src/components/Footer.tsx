import React from "react";
import styles from "./footer.scss";

interface Props {}

const Footer: React.FC<Props> = () => {
  return (
    <div className={styles.footer}>
      <p>
        API provided by{" "}
        <a href="https://www.themoviedb.org/" target="_blank">
          TMDb
        </a>
      </p>
    </div>
  );
};

export default Footer;
