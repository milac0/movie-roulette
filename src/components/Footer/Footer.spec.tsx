import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './Footer';

describe('Footer renders correctly', () => {
    test('snapshot renders', () => {
        const tree = renderer.create(<Footer />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});

