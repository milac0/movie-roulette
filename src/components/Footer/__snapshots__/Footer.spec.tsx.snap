// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly snapshot renders 1`] = `
<div
  className="footer"
>
  <p>
    API provided by
     
    <a
      href="https://www.themoviedb.org/"
      target="_blank"
    >
      TMDb
    </a>
  </p>
</div>
`;
